1001-2000/LIT1455GadlaG.xml

```diff
@@ -51,52 +61,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-               <relation name="betmas:isAuthorOfEthiopicTranslation" active="LIT1455GadlaG" passive="PRS7057Michael"></relation>
-               <relation name="saws:isAttributedToAuthor" active="LIT1455GadlaG" passive="PRS13151Saqrates"></relation>
+               <relation name="betmas:isAuthorOfEthiopicTranslation" active="LIT1455GadlaG"
+                  passive="PRS7057Michael"/>
+               <relation name="saws:isAttributedToAuthor" active="LIT1455GadlaG"
+                  passive="PRS13151Saqrates"/>
             </listRelation>
          </div>
          <div type="edition" xml:lang="gez">
-            <div type="textpart" subtype="incipit"><ab> ገድል፡ ወስምዕ፡ ዘቅዱስ፡ ወብፁዕ፡ ሰማዕት፡
-               ዐቢይ፡ ማር፡ ጊዮርጊስ፡   etc. ዘፈጸመ፡ ገድሎ፡ አመ፡ ፪ወ፫፡ ለወርኃ፡ መርሙደ፡ ዘውእቱ፡ ሚያዝያ።</ab></div>
-            <div type="textpart" subtype="colophon" xml:id="coloph1"><ab>ወሶበ፡ ፈጸመ፡ ቅዱስ፡ ጊዮርጊስ፡ ገድሎ፡ አነ፡ ሰቅራጥስ፡ ገብሩ፡ ለጊዮርጊስ፡ ኮንኩ፡ አትልዎ፡ ለእግዚእየ፡ ጊዮርጊስ፡ በኵሉ፡
-               {ገድሉ}፡ ወፃማሁ፡ ወለካልሁ፡ ዘንተ፡ ጸሐፍኩ፡ … ተፈጸመ፡
-               እንዘ፡ ንነግድ፡ ብሔረ፡ ቈስጠንጢኖስ፡ መካን፡ ዘስሙ፡ ቍስቋስ፡ መርስ፡ እንዘ፡ ንነግድ፡ ወአስተዳሎኩ፡ ዘንተ፡ መጽሐፈ።
-               ወኮነ፡ በዘአዕረፈ፡ እግዚእየ፡ ጊዮርጊስ፡ ሃብሩ፡ ለኢየሱስ፡ ክርስቶስ፡ በዘተከለለ፡ ወርኁ፡ በጽርዕ፡ ጵንፍልዮ፡ ስሙ፡
-               ወትርጓሜሁ፡ ወርኃ፡ ሚያዝያ፡ አመ፡ ፳ወ፫ለሠርቅ፡ ወዕለቱሂ፡ ዓርብ፡ በጊዜ፡ ፱ሰዓት፡ አሜን። ሃሌ፡ ሉያ፡ ተተርጐመት፡ ዛቲ፡ መጽሐፍ፡ አምሳነ፡ ዓረቢ፡ በግዕዝ፡ በአፈ፡ ሚካኤል፡ {ዘይንዕስ} ፡ ወልደ፡ ወልዱ፡
-                  ለአባ፡ ጊዮርጊስ፡ ወልደ፡ አባ፡ ሚካኤል፡ ጳጳስ። … …
-                  ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን፡ ለይኩን፡ ለይኩን፡</ab><note>quoted after <ref type="mss" corresp="ESaqg014"></ref>, same in <bibl><ptr target="bm:Budge1930George"/><citedRange>274-276</citedRange></bibl></note></div>
-            <div type="textpart" subtype="colophon" xml:id="coloph2"><ab>ሃሌ፡ ሉያ፡ ተተርጐመት፡ ዛቲ፡ መጽሐፍ፡ እምልሳነ፡ ዐረቢ፡ ለግዕዝ፡ በአፈ፡ 
-                  <persName ref="PRS12752Mikael">ሚካኤል፡</persName>  ዘይንስ፡
-                  ወልደ፡ ወልዱ፡ ለ<persName ref="PRS12753Giyorgis"><roleName type="title">አባ፡</roleName>ጊዮርጊስ፡</persName> 
-                  ወልደ፡ አባ፡ <persName ref="PRS12754Mikale"><roleName type="title">አባ፡</roleName> ሚካኤል፡</persName> ጳጳስ፡ እንዘ፡ ይረድኦ፡ መንፈስ፡ ቅዱስ።</ab>
-               <note>Copied colophon of the translation, taken from <ref type="mss" corresp="BLorient712"></ref></note>
-               <div type="textpart" subtype="colophon" xml:id="coloph3"><ab> ወፍዳሜሁሰ፡ ኮነ፡፪ለወርኃ፡ ታስሪን፡ ቀደማይ፡ በሐሳቦሙ፡ ለዕብራውያን፡ ወበግዕዝ፡ አመ፡፱ዓውድ፡ ቀመር፡ ነ፩ጥንትዮን፡ ወበ፱ዓውደ፡ ዕለት። በ፱አበቅቴ፡ በካልአ፡ ዓመተ፡ መንግሥቱ፡ ለደዊት፡ ንጉሠ፡ አግዓዜ፡ ወልደ፡ ናዖድ፡ ንጉሠ፡ ድቱዓ፡ ሃይማኖት፡ በዕለተ፡ ሰኑይ፡ በ፲ሰዓት፡ ዘመዓልት፡ አመ፡ ከርመ፡ በምድረ፡ አዘዞ።
-               </ab>
-                   <note>The copied colophon (taken from <ref type="mss" corresp="BLorient713"></ref>) gives the precise date of translation, <date when="1510">2nd year of the reign of Dawit (that is, Lǝbna Dǝngǝl), who began his reign in August 1508</date> </note>
+            <div type="textpart" subtype="incipit">
+               <ab> ገድል፡ ወስምዕ፡ ዘቅዱስ፡ ወብፁዕ፡ ሰማዕት፡ ዐቢይ፡ ማር፡ ጊዮርጊስ፡ etc. ዘፈጸመ፡ ገድሎ፡ አመ፡ ፪ወ፫፡ ለወርኃ፡
```

Comment:
I think "etc." should be encoded differently. In the guidelines (https://betamasaheft.eu/Guidelines/?q=erased&id=text-encoding) the use of `<gap reason="ellipsis"/>` is recommended for portions of text omitted from the edition by the editor, for whatever reason (brevity, context, language, etc.)

Reply:
This was already in the entry. I simply added the bibliographic references. I decided to restrict myself to what we need in our catalogue, this are just the refernces. Therefore, I left the encoding as it was.

Reply:
Sorry, I thought the entire encoding was yours.